content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,320 @@
+- :guilabel:`Average Subcontracting Cost/Unit`
+- :guilabel:`Quantity Produced`
+
+To change the data categories displayed, click the :guilabel:`Measures` :icon:`fa-caret-down`
+button, and select or deselect categories from the popover window that appears.
+
+Click one of the column titles to sort the data from the lowest to the highest value. Click the
+title again to sort from the highest to the lowest value.
+
+To make the rows of the pivot table display time periods instead of products, begin by clicking the
+:icon:`fa-minus-square-o` :guilabel:`Total` button above the product names, which removes the
+product rows, and changes the button to appear as :icon:`fa-plus-square` :guilabel:`Total`.
+
+Click :icon:`fa-plus-square` :guilabel:`Total` to open a popover window showing options for the data
+categories that can be displayed. Mouse over the :guilabel:`End Date` line to open a second popover
+window showing time periods to choose from. Click one of the time periods, and the table updates to
+show one unit of the period for each row.
+
+.. image:: production_analysis/pivot-view.png
+   :align: center
+   :alt: The pivot view of the Production Analysis report.
+
+Use case: compare products
+==========================
+
+One of the best uses for the production analysis report is comparing statistics about two or more
+products. This is accomplished by entering the products into the :guilabel:`Search...` bar, then
+selecting the necessary measure, filter, and grouping, to see the desired data.
+
+.. example::
+   Toy manufacturer *Tommy's Toys* is trying to reduce their manufacturing operation costs. To
+   accomplish this, they've decided to identify redundant products and cease manufacturing the ones
+   with higher operation costs.
+
+   Two of the toys that have been singled out for analysis are the *pogo stick* and *moon shoes*.
+   Tommy's Toys believes that these two toys are so similar that they can stop manufacturing one,
+   without significantly impacting their product offering.
+
+   To compare operation costs for the toys, business analyst Mike opens the
+   :guilabel:`Manufacturing` app and navigates to the :guilabel:`Production Analysis` page. In the
+   :guilabel:`Search...` bar, he enters the names of both products. Then, he opens the drop-down
+   menu, and clicks :guilabel:`Product` in the :guilabel:`Group By` section.
+
+   Below the :guilabel:`Search...` bar, Mike clicks on the :guilabel:`Measures` :icon:`fa-caret-down`
+   drop-down menu, and selects the :guilabel:`Total Operation Cost/Unit` option. Finally, he selects
+   the :icon:`fa-bar-chart` :guilabel:`(bar chart)` graph type.
+
+   With these options selected, the production analysis report shows a bar chart for the current
+   year, with one bar for each product, signifying the average operation cost for one unit of the
+   product.
+
+   With this data, Mike is able to see that the average operation cost for the moon shoes is almost
```

Comment:
I know you've told me that certain formatting decisions have been made in the past with example admonitions like this that repeatedly mention a company or product, but I *do*, personally, think it looks better if, at the very least, the product name is capitalized... but, since you've said that you've been told to do otherwise, I guess this is just a pure suggestion to consider...